docs/readmes/lte/deploy_install_docker.md

```diff
@@ -48,11 +48,10 @@ installation process to get an IP using DHCP.
 
 #### Do pre-installation steps
 
-Become root user and switch directory:
+Become root user:
 
 ```bash
-sudo su
-cd /root
+sudo su - root
```

Comment:
`sudo su - root` is equivalent to `sudo -i` which is recomended AFAIK.

Reply:
Updated this to `sudo -i`